modules/dxp/apps/saml/saml-persistence-api/src/main/java/com/liferay/saml/persistence/service/SamlPeerBindingLocalServiceWrapper.java

```diff
@@ -0,0 +1,382 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: